### DISCUSSION TITLE
Broken package URL falsely reported as OK by github-actions bot

### DISCUSSION THREAD

**Discussion by dbarbuzzi** [ScoopInstaller/Scoop#5662] [Help and Questions]
I recently tried reporting an issue with the `main/brotli` package, which objectively has bad URLs for both versions of the package in its JSON file (tested with multiple download tools across multiple machines across multiple networks/geographical locations). Most details are in https://github.com/ScoopInstaller/Main/issues/5136 ([a comment I added there](https://github.com/ScoopInstaller/Main/issues/5136#issuecomment-1737889685) links the file as well as extracts both URLs for easy reproduction).

However, the github-actions bot is claiming that it is able to download the files. Given I tried across two different geographical locations, it seems more likely an issue with the bot (perhaps it inadvertently has a local cache layer somewhere along its infrastructure?).

**Comment by rashil2000** [comment]
The URLs have been updated: https://github.com/ScoopInstaller/Main/commit/9308e2a16b14fd76c795f3b0b538b9f7a4f9c8e4
